File: Codewars/Javascript/7 kyu/II/Lost number in number sequence.js
function findDeletedNumber(arr, mixArr) {
    for(let i=0; i<arr.length; i++)
        if(!mixArr.includes(arr[i])) return arr[i]
    return 0
}

const findDeletedNumberUp = (arr, mixArr) => arr.filter(e => mixArr.indexOf(e)==-1)[0] || 0

console.log(findDeletedNumber([1,2,3,4,5], [3,4,1,5]));
console.log(findDeletedNumberUp([1,2,3,4,5], [3,4,1,5]));

File: Codewars/Python/7 kyu/III/Lost number in number sequence.py
def find_deleted_number(arr, mixed_arr):
    for i in arr:
        if(i in mixed_arr):
            pass
        else:
            return i
    return

def find_deleted_number_up(arr, mixed_arr):
    return sum(arr)-sum(mixed_arr)

def find_deleted_number_up2(arr, mixed_arr):
    for i in arr:
        if i not in mixed_arr:
            return i
    return 0

print(find_deleted_number([1,2,3,4,5], [3,4,1,5]))
print(find_deleted_number_up([1,2,3,4,5], [3,4,1,5]))
print(find_deleted_number_up2([1,2,3,4,5], [3,4,2,1,5]))